### ISSUE TITLE
Graph links do not meet the color-contrast ratio in dark theme

### ISSUE THREAD

**Issue by ElinorW** [microsoftgraph/microsoft-graph-explorer-v4#707]
**Describe the bug**
The colour used on the links has a low colour contrast ratio, hence affecting visibility when in dark mode.

**To Reproduce**

**Expected behavior**
The Colour contrast ratio should be least 4.5:1 to ensure clear visibility

**Screenshots**
![image](https://user-images.githubusercontent.com/18407044/94003563-0fd06180-fda4-11ea-99fd-466609efa1b2.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Edge
 - Version: 87.0.637.0 (Official build) dev (64-bit)

**Smartphone (please complete the following information):**

**Additional context**
 
[AB#7462](https://microsoftgraph.visualstudio.com/0985d294-5762-4bc2-a565-161ef349ca3e/_workitems/edit/7462)
